<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>921</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Nouvel accuse</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Pr√©nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_prenom">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nom">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>200</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>type</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_type">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tab_accuse">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>140</y>
        <width>451</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id_supp">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>290</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>290</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>250</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer accuse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>40</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher accuse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_modifier">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>200</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>recuperer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_recherche">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>110</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="recherche">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>110</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>recherche</string>
      </property>
     </widget>
     <widget class="QComboBox" name="rechercher">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>110</y>
        <width>73</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>id</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>prenom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>type</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="trie">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>80</y>
        <width>73</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>id</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>prenom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>type</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>80</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>sort by</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>340</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>EXCEL</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>nouveau enquete</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>idm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>description</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>typem</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_idm">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_description">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_typem">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>200</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_enquete">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>100</y>
        <width>431</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_idm_supp">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>290</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer_1">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>290</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>idm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>260</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer enquete</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>30</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher enquete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>200</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>recuperer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="recherche1">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>70</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>recherche</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_recherche1">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>70</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="trie_2">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>40</y>
        <width>73</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>idm</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>description</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>typem</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="rechercher_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>70</y>
        <width>73</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>idm</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>description</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>typem</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>40</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>sort by</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>330</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>EXCEL</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>accuse statistique</string>
     </attribute>
     <widget class="QCustomPlot" name="customPlot" native="true">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>661</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>enquette statistique</string>
     </attribute>
     <widget class="QCustomPlot" name="customPlot_2" native="true">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>661</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>969</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
